master/internal/api_workspace.go

```diff
@@ -66,36 +67,46 @@ func maskStorageConfigSecrets(w *workspacev1.Workspace) error {
 	return nil
 }
 
-func (a *apiServer) validateRequestClusterName(clusterName string) error {
+func (a *apiServer) validateRequestClusterName(clusterName string) (string, error) {
 	if len(clusterName) == 0 && len(a.m.allRms) > 1 {
-		return status.Errorf(codes.InvalidArgument, "must specify a cluster name when using MultiRM")
+		return "", status.Errorf(codes.InvalidArgument, "must specify a cluster name when using MultiRM")
+	}
+
+	if len(a.m.allRms) == 1 && clusterName == "" {
```

Comment:
Can we leave a comment above this if-block that's something along the lines of
`// Since SingleRM clusters don't have to specify the cluster name in namespace-binding requests, we populate the cluster name appropriately if it is set in the master config`.

Reply:
done